forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -769,7 +769,11 @@ public Optional<HostUser> closedBy() {
                       .map(JSONValue::asObject)
                       .filter(obj -> obj.contains("event"))
                       .filter(obj -> obj.get("event").asString().equals("closed"))
-                      .reduce((a, b) -> b)
+                      .max((first, second) -> {
+                          var firstTime = ZonedDateTime.parse(first.get("created_at").asString());
+                          var secondTime = ZonedDateTime.parse(second.get("created_at").asString());
+                          return firstTime.compareTo(secondTime);
+                      })
```

Comment:
```suggestion
                      .max(Comparator.comparing(o -> ZonedDateTime.parse(o.get("created_at").asString())))
```

Reply:
It will parse the date up to two times for each element, but I think that's ok. There shouldn't be that many of them.

Reply:
Fixed.